src/engine/sys/sdl_glimp.cpp

```diff
@@ -2202,9 +2208,36 @@ static void GLimp_InitExtensions()
 	// made required in OpenGL 4.2
 	glConfig2.shaderAtomicCountersAvailable = LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, ARB_shader_atomic_counters, r_arb_shader_atomic_counters.Get() );
 
+	// made required in OpenGL 4.6
+	glConfig2.shaderAtomicCounterOpsAvailable = LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, ARB_shader_atomic_counter_ops, r_arb_shader_atomic_counter_ops.Get() );
+
 	// not required by any OpenGL version
 	glConfig2.indirectParametersAvailable = LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, ARB_indirect_parameters, r_arb_indirect_parameters.Get() );
 
+	// not required by any OpenGL version
+	glConfig2.shaderSubgroupAvailable = LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, KHR_shader_subgroup, r_khr_shader_subgroup.Get() );
+
+	if ( glConfig2.shaderSubgroupAvailable ) {
+		int subgroupFeatures;
+		glGetIntegerv( GL_SUBGROUP_SUPPORTED_FEATURES_KHR, &subgroupFeatures );
+
+		glConfig2.shaderSubgroupBasicAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_BASIC_BIT_KHR;
+		glConfig2.shaderSubgroupVoteAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_VOTE_BIT_KHR;
+		glConfig2.shaderSubgroupArithmeticAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_ARITHMETIC_BIT_KHR;
+		glConfig2.shaderSubgroupBallotAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_BALLOT_BIT_KHR;
+		glConfig2.shaderSubgroupShuffleAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_SHUFFLE_BIT_KHR;
+		glConfig2.shaderSubgroupShuffleRelativeAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_SHUFFLE_RELATIVE_BIT_KHR;
+		glConfig2.shaderSubgroupQuadAvailable = subgroupFeatures & GL_SUBGROUP_FEATURE_QUAD_BIT_KHR;
+
+		const char* trueFalse[2] = { "false", "true" };
+
+		Log::Notice( "Supported subgroup extensions: basic: %s, vote: %s, arithmetic: %s, ballot: %s, shuffle: %s, "
+			"shuffle_relative: %s, quad: %s", trueFalse[glConfig2.shaderSubgroupBasicAvailable],
+			trueFalse[glConfig2.shaderSubgroupVoteAvailable], trueFalse[glConfig2.shaderSubgroupArithmeticAvailable],
```

Comment:
With tinyformat you use `%s` with a bool you automatically get the boolalpha word version. Although yes/no would sound better IMO

Reply:
Oh, I thought it just did the same thing as `printf()`.